src/ResourceManagement/Search/AdminKeysImpl.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+namespace Microsoft.Azure.Management.Search.Fluent
```

Comment:
@martinsawicki might be able to talk more about the future of Fluent branch but as of now the code stays in a separate branch out of the Azure/azure-sdk-for-net repo.